app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/PriceChange.kt

```diff
@@ -0,0 +1,63 @@
+package co.nimblehq.compose.crypto.ui.screens.home
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.KeyboardArrowDown
+import androidx.compose.material.icons.filled.KeyboardArrowUp
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.theme.Color
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp8
+import co.nimblehq.compose.crypto.ui.theme.Style
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun PriceChange(
+    modifier: Modifier,
+    isPositiveNumber: Boolean = false
+) {
+    Row(modifier = modifier) {
+        Icon(
+            imageVector = if (isPositiveNumber) {
+                Icons.Filled.KeyboardArrowUp
+            } else {
+                Icons.Filled.KeyboardArrowDown
```

Comment:
I think we're not using correct icons from the design 🥲 

![image](https://user-images.githubusercontent.com/16315358/186855283-3cec2c01-507a-4cce-88de-7b28c41abd98.png)

vs

![image](https://user-images.githubusercontent.com/16315358/186855367-dd2a04a6-e68a-4b8c-8644-3d68566d0d4c.png)